circle.yml

```diff
@@ -1,13 +1,19 @@
 dependencies:
+  pre:
+    - curl -o- -L https://yarnpkg.com/install.sh | bash
```

Comment:
`--location` may not currently be necessary but I think it's a good idea to include it. If the canonical location of the file were to change, and the current location were to redirect to the new location, it would be nice for the Yarn installation to continue to work.